en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,159 @@
+# Holybro X500 V2 + Pixhawk 6C (PX4 Dev Kit)
+
+This topic provides full instructions for building the [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html), also known as the Holybro PX4 Dev Kit.
+
+![The fully built vehicle with props removed](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/kit_no_props.jpg)
+
```

Comment:
[Previous version](http://docs.px4.io/main/en/frames_multicopter/holybro_x500V2_pixhawk5x.html) had a lot more text here, including introduction Bill of materials, key information, hardware, electronics. Do you plan on adding that?

Reply:
As discussed these are useful. Why? Because they allow users to check they have everything needed before they start. They also identify any bits you have to purchase that may no longer be in the kit.

That said, right now they are not essential, and they are less essential for something like this which is delivered as a kit by a reliable vendor.